GOVERNANCE.rst

```diff
@@ -0,0 +1,183 @@
+sktime tools. You can implement your favorite algorithm in a sktime
+compatible way in one of `our companion repositories <https://github
+.com/sktime>`_ on GitHub. We will be happy to list it under `related
+software <https://github.com/alan-turing-institute/sktime/wiki/related
+-software>`_.
+
+If algorithms require major dependencies, we encourage to create a separate
+companion repository. For example, for deep learning techniques based on
+TensorFlow and Keras, we have `sktime-dl <https://github
+.com/sktime/sktime-dl>`_. For smaller dependencies which are limited to a few files, we encourage to use soft dependencies, which are only required for particular modules, but not for most of sktime's functionality and not for installing sktime.
+
+If significant issues are not fixed by the maintainer, and no other
+contributor volunteers to fix the algorithm within 3 months, we will
+deprecate and remove the algorithm from sktime eventually.
+
+Decision making
+---------------
+
+The purpose of this section is to formalize the decision-making process used by the sktime project. We clarify how decisions are made and who can make them.
+
+sktime's decision-making process is designed to take into account feedback from all community members and strives to find consensus, while avoiding deadlocks when no consensus can be found.
+
+Decisions about the future of the project are made through discussion with
+all members of the community. All discussion takes place on the project’s
+`issue tracker <https://github.com/alan-turing-institute/sktime/issues>`_ or
+pull requests. Sensitive discussion can occur on private chats.
+
+sktime uses a "consensus seeking" process for making decisions. The community tries to find a resolution that has no open objections among core developers.
+
+Roles
+~~~~~
+We differentiate between three roles:
+* Contributors
+* Core developers
+* Technical committee members
+
+Contributors
+++++++++++++
+
+Contributors are community members who contribute in concrete ways to the project. Anyone can become a contributor, and contributions can take many forms – not only code – as detailed in the contributing guide.
+
+Core developers
++++++++++++++++
+
+Core developers are community members who have shown that they are dedicated to the continued development of the project through ongoing engagement with the community. They have shown they can be trusted to maintain sktime with care.
+
+* **Direct access.** Being a core developer allows contributors to more easily carry on with their project related activities by giving them direct access to the project’s repository.
+* **Issue/PR management.** Core developers can review and manage issues and pull requests. This includes commenting on issues, reviewing code contributions, merging approved pull requests, and closing issues once resolved.
+* **Voting.** They can cast votes for and against merging a pull-request, and can be involved in deciding major changes to the API.
```

Comment:
somewhat related to governance, but more a tangential topic: is the gitflow model written down somewhere?
That should maybe also mirror and refer to the different roles.

Reply:
I've just updated the CONTRIBUTING.rst a few days ago, in what way should that refer to the different roles?